### PR TITLE
Create second button

### PR DESCRIPTION
**Pull request by Onsshuk2**

### CODE REVIEW

**Comment by Grinchek** [comment]
Very good

**Comment by Mishkanchik** [comment]
nice glhf

File: Client_App/MainWindow.xaml.cs [hunk 1]
```diff
@@ -29,5 +29,10 @@ private void Button_Click(object sender, RoutedEventArgs e)
         {
             MessageBox.Show("You clicked me");
         }
+
+        private void Button_Click_1(object sender, RoutedEventArgs e)
```
Comment (Mishkanchik) [inline Client_App/MainWindow.xaml.cs:33]: good luck

**Comment by Grinchek** [comment]
Please , fix , anything

**Review by Mishkanchik — APPROVED** [review]
nice ### `_glhf****_`

**Review by Grinchek — COMMENTED** [review]
Ok

**Review by Mishkanchik — APPROVED** [review]

**Review by Grinchek — APPROVED** [review]

**Review by Mishkanchik — APPROVED** [review]
da aperole

File: Client_App/MainWindow.xaml.cs (hunk 1 above)
Comment (Onsshuk2) [reply Client_App/MainWindow.xaml.cs:33]: oki

**Pull request merged**